spoolman/database/database.py

```diff
@@ -197,6 +209,9 @@ def schedule_tasks(scheduler: Scheduler) -> None:
         logger.info("Scheduling automatic database backup for midnight.")
         # Schedule for midnight
         scheduler.daily(datetime.time(hour=0, minute=0, second=0), _backup_task)  # type: ignore[arg-type]
+        logger.info("Scheduling automatic metric collection.")
+        logger.info("%s", datetime.time(minute=1))
+        scheduler.minutely(datetime.time(second=0), _metrics)
```

Comment:
This will now only get run if automatic backups are enabled and the database is a sqlite one, which is irrelevant of metrics collection.